help/en/releasenotes/current-draft-note.shtml

```diff
@@ -687,4 +687,5 @@
         <ul>
             <li>The pop-up menu in the various Tables now has an entry
                 for "Copy System Name".</li>
+            <li>Add new editor panel to the Editor Manager when doing a CPE/PE switch.</li>
```

Comment:
A suggestion is to add "Bug fix:" to make it more clear why this change is done.

`<li>Bug fix: Add new editor panel to the Editor Manager when doing a CPE/PE switch.</li>`